cf_remote/remote.py

```diff
@@ -278,7 +287,9 @@ def install_package(host, pkg, data, *, connection=None):
                 else "--allow-unsigned-rpm"
             )
             output = ssh_sudo(
-                connection, "zypper install -y {} {}".format(allow_unsigned, pkg), True
+                connection,
+                "zypper install -y {} {}".format(allow_unsigned, pkg),
+                True,
```

Comment:
This is an unrelated change.

Reply:
right, necessary black formatting, I can place in a separate commit